test/functional/normalization/multiline_exception_test.go

```diff
@@ -85,8 +159,12 @@ created by main.main
 			crioLine := functional.NewCRIOLogMessage(timestamp, line, false)
 			buffer = append(buffer, crioLine)
 		}
-		// Application log in namespace
-		Expect(framework.WriteMessagesToNamespace(strings.Join(buffer, "\n"), appNamespace, 1)).To(Succeed())
+
+		if testfw.LogCollectionType == logging.LogCollectionTypeVector {
+			Expect(framework.WriteMessagesToApplicationLog(strings.Join(buffer, "\n"), 1)).To(Succeed())
+		} else {
+			Expect(framework.WriteMessagesToNamespace(strings.Join(buffer, "\n"), appNamespace, 1)).To(Succeed())
```

Comment:
This works for vector as long as it's the same ns as the test pod. And then maybe you could enable the other tests

Reply:
> This works for vector as long as it's the same ns as the test pod. And then maybe you could enable the other tests

I checked, these tests not work for vector without modification related to the namespace name